assets/Makefile

```
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2023-2025 Nomadic Labs <contact@nomadic-labs.com>
+# SPDX-FileCopyrightText: 2024-2025 TriliTech <contact@trili.tech>
+#
+# SPDX-License-Identifier: MIT
+
+.PHONY: update-assets
+update-assets: riscv-dummy.elf jstz
+
+riscv-dummy.elf riscv-dummy.elf.checksum::
+	@make -C .. dummy/build
+	@cp ../kernels/dummy/target/riscv64gc-unknown-linux-musl/release/riscv-dummy $@
+	@sha256sum $@ > $@.checksum
+
+jstz jstz.checksum::
```

Neither of these rules require double colon.

```suggestion
riscv-dummy.elf riscv-dummy.elf.checksum:
	@make -C .. dummy/build
	@cp ../kernels/dummy/target/riscv64gc-unknown-linux-musl/release/riscv-dummy $@
	@sha256sum $@ > $@.checksum

jstz jstz.checksum:
```

---

Updated [here](https://github.com/tezos/riscv-pvm/compare/0a7eb8c252f229af3241d0c1d949d138c939c198..51104f9e6d85a3ac3c55475e3de4979fa4321de2). Not sure why, but the double colon made them also behave like `.PHONY` rules, I had to mark them explicitly when I removed it.